core/src/main/scala/com/avast/clients/rabbitmq/DefaultRabbitMQClientFactory.scala

```diff
@@ -106,7 +106,6 @@ private[rabbitmq] object DefaultRabbitMQClientFactory extends LazyLogging {
       connectionInfo: RabbitMQConnectionInfo,
       blocker: Blocker,
       monitor: Monitor)(implicit cs: ContextShift[F]): DefaultRabbitMQProducer[F, A] = {
-    import producerConfig._
```

Comment:
I've removed it as this doesn't give us much benefit but makes us confused with very common field names like `name` in it.